final_task/pycalc/rpn.py

```diff
@@ -0,0 +1,116 @@
+"""This module contains a class that allows to transform infix notation math tokens into RPN"""
+
+# import from pycalc self library
+from .pycalclib import constants, negative_constants, operators, comparison_operators, precedence
+from .pycalclib import functions, negative_functions
+
+
+class RPN():
+    """A model of rpn capable of converting infix to postfix (RPN) notation"""
+
+    def __init__(self, tokens):
+        """Initialize rpn"""
+        self.tokens = tokens
+        self.operators_stack = []
+        self.output_queue = []
+        self.error_msg = None
+        self.constants = constants
+        self.negative_constants = negative_constants
+        self.operators = operators
+        self.comparison_operators = comparison_operators
+        self.precedence = precedence
+        self.functions = functions
+        self.negative_functions = negative_functions
+
+    def is_left_associative(self, operator):
+        """Determines whether operator is left associative"""
+        if operator in ['^', '**']:
+            return False
+        else:
+            return True
+
+    def is_number(self, token):
```

Comment:
There is the same method in `Multsignsadder`. You can move this function into `utils.py` module and reuse it.

Reply:
moved to utils.py as a function